resources/Rust.opj

```diff
@@ -1,6 +1,6 @@
 {
   "Name": "Rust",
-  "TargetDirectory": "D:\\Servers\\Rust\\RustDedicated_Data\\Managed",
+  "TargetDirectory": "C:\\Users\\2CHEVSKII\\Desktop\\GameServers\\rust-ds\\RustDedicated_Data\\Managed",
```

Comment:
You ought not change the TargetDirectory.  We cannot accept a PR directly with path changes like this.

Reply:
Left it by accident, but you could change it manually. Anyways - gonna make changes myself in a moment.